modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/LiveSchemaChangeTableTest.java

```diff
@@ -0,0 +1,177 @@
+import org.apache.ignite.internal.table.ColumnNotFoundException;
+import org.apache.ignite.internal.table.TableImpl;
+import org.apache.ignite.table.KeyValueBinaryView;
+import org.apache.ignite.table.Table;
+import org.apache.ignite.table.Tuple;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/**
+ * Live schema tests.
+ */
+@Disabled("https://issues.apache.org/jira/browse/IGNITE-14581")
+class LiveSchemaChangeTableTest extends AbstractSchemaChangeTest {
+    /**
+     * Check live schema add columns
+     */
+    @Test
+    public void testLiveSchemaAddColumns() {
+        List<Ignite> grid = startGrid();
+
+        createTable(grid);
+
+        Table tbl = grid.get(1).tables().table(TABLE);
+
+        ((TableImpl)tbl).schemaType(SchemaType.LIVE_SCHEMA);
+
+        Tuple row = tbl.tupleBuilder().set("key", 1L).set("valStrNew", "111").set("valIntNew", 333).build();
+
+        tbl.insert(row);
+
+        Tuple res = tbl.get(row);
+
+        assertEquals("111", res.value("valStrNew"));
+        assertEquals(Integer.valueOf(333), res.value("valIntNew"));
+
+        Tuple secondRow = tbl.tupleBuilder().set("key", 2L).set("valStrNew", "222").set("valIntNew", 42).build();
+
+        tbl.insert(secondRow);
+
+        Tuple res2 = tbl.get(secondRow);
+
+        assertEquals("222", res2.value("valStrNew"));
+        assertEquals(Integer.valueOf(42), res2.value("valIntNew"));
+    }
+
+    /**
+     * Check live schema kvBinaryView add columns
+     */
+    @Test
+    public void testLiveSchemaAddColumnsKVBinaryView() {
+        List<Ignite> grid = startGrid();
+
+        createTable(grid);
+
+        Table tbl = grid.get(1).tables().table(TABLE);
+
+        ((TableImpl)tbl).schemaType(SchemaType.LIVE_SCHEMA);
+
+        KeyValueBinaryView kvBinaryView = tbl.kvView();
+
+        Tuple key = kvBinaryView.tupleBuilder().set("key", 1L).build();
+        Tuple val = kvBinaryView.tupleBuilder().set("valStrNew", "111").set("valIntNew", 333).build();
+
+        kvBinaryView.put(key, val);
+
+        Tuple key2 = kvBinaryView.tupleBuilder().set("key", 2L).build();
+        Tuple val2 = kvBinaryView.tupleBuilder().set("valStrNew", "222").set("valIntNew", 42).build();
+
+        kvBinaryView.put(key2, val2);
+
+        Tuple res = kvBinaryView.get(key);
+        assertEquals("111", res.value("valStrNew"));
+        assertEquals(Integer.valueOf(333), res.value("valIntNew"));
+
+
+        Tuple res2 = kvBinaryView.get(key2);
+
+        assertEquals("222", res2.value("valStrNew"));
+        assertEquals(Integer.valueOf(42), res2.value("valIntNew"));
+    }
+
+    /**
+     * Check live schema tuple can handle different value types
+     */
+    @Test
+    public void testLiveSchemaDifferentColumnTypes() {
```

Comment:
I'd check all new columns are nullable and 'null' is a default value.
Note that 'null' can't be the default value for the string for now, due to a bug (config doesn't support nulls).